recipes/sagemath-db-conway-polynomials/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+curl -L -O https://raw.githubusercontent.com/sagemath/sage/7.5.1/build/pkgs/conway_polynomials/spkg-install
```

Comment:
I didn't want to copy the build script when I could just download it and use it.
I'll add a note in the feedstock